kobodl/kobo.py

```diff
@@ -496,4 +526,19 @@ def Login(self, email: str, password: str, captcha: str) -> None:
         ]  # We don't call self.Settings.Save here, AuthenticateDevice will do that if it succeeds.
         userKey = parsedQueries["userKey"][0]
 
+        cookie = SimpleCookie()
+        cookie.load(authCookie)
+
+        def get_cookie_value(cookie_name):
+            if cookie_name in cookie:
+                return cookie[cookie_name].value
+            return None
```

Comment:
This closure is never called; did you mean to leave this in here?

Reply:
Nice catch! That definitely shouldn't be in here. I'll delete it